docs/abbr.md

```diff
@@ -53,7 +54,7 @@ Note that there are always exceptions, especially when we try to comply with the
 |                  | input / output                      | io             |                                                  |
 |                  | object                              | obj            |                                                  |
 |                  | string                              | s              |                                                  |
-|                  | class                               | cls            |                                                  |
+|                  | class                               | cl             | cl, classes                                      |
```

Comment:
This should still be shown as `cls` for classmethod 1st param right?

Reply:
You suggested cl, and classes here: 
http://forums.fast.ai/t/abbreviations-for-train-validation-test-variables/19618/31
we agreed that both cls and class shouldn't be used as they are reserved words.

Reply:
Sorry if I didn't explain properly; yes `cl` and `classes` are correct for that purpose, but `cls` is not a reserved word - it is simply the (by convention) name of the first parameter to a classmethod in python. I'm just wondering if we should explicitly mention this usage here.